README.md

```diff
@@ -1,16 +1,89 @@
-# Title
+# Program With Expressions
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Write a program with conditional logic using evaluation expressions only
 
 ## Introduction
 
-## SWBAT 1
+You've traveled quite a road! You started off, from a conversational
+perspective, mute and unable to interact, and now you can converse with Ruby.
 
-## SWBAT 2
+Not only that, but you can ask Ruby to respond with some very sophisticated
+results! Let's prove it to ourselves! But take this as an important fact:
+
+***Programming is the art of helping a computer decide which expressions to
+evaluate and in which order.***
+
+Believe it or not, the est of your programming career will be learning to make
+your messages to Ruby, and other programmers, as clear and efficient as
+possible. That's the name of the game for this career ***and you're doing it
+now!***  
+
+## Write a Program with Conditional Logic Using Evaluation Expressions Only
+
+Recall when we introduced the ternary expression, we wrote this little program:
+
+```ruby
+likely_to_rain = true
+garment = likely_to_rain ? "galoshes" : "sun hat"
+```
+
+Let's expand on this idea and use our new-found knowledge of `String` to make
+the output more informative. You will see many of the expressions you learned in
+this module come into play. We're going to mix-and-match techniques to show you
+just how much you've learned!
+
+```ruby
+name = "Your name here"
+rain_percentage = 0.2
+temperature_in_c = 26
+
+likely_to_rain = rain_percentage > 0.30
+sun_is_dangerous = temperature_in_c >= 26
+
+"Hello, #{name}, with a rain chance of #{rain_percentage * 100}% and a temperature of #{temperature_in_c}C we recommend that you " + (likely_to_rain ? "take an umbrella" : "enjoy this rain-free day") +
```

Comment:
one question about this - why not break the ternaries out into variable assignments first?

```
name = "Your name here"
rain_percentage = 0.2
temperature_in_c = 26
 likely_to_rain = rain_percentage > 0.30
sun_is_dangerous = temperature_in_c >= 26

weather_message = likely_to_rain ? "take an umbrella" : "enjoy this rain-free day"
heat_message = sun_is_dangerous ? ' and watch out for heat stroke!' : ' and fine weather.'

"Hello, #{name}, with a rain chance of #{rain_percentage * 100}% and a temperature of #{temperature_in_c}C we recommend that you " + weather_message + heat_message
```

Or maybe both can be shown if you think one is clearer than the other?